lib/rails/dom/testing/assertions/selector_assertions/html_selector.rb

```diff
@@ -14,11 +14,11 @@ class HTMLSelector # :nodoc:
 
             include Minitest::Assertions
 
-            def initialize(values, previous_selection = nil, &root_fallback)
+            def initialize(values, previous_selection = nil, refute: false, &root_fallback)
```

Comment:
I really wanted to avoid this API, but the dynamic way in which `assert_dom` accepts arguments meant that the refute conditions needed to be handled in the selector, as opposed to simply delegating to `assert_dom` with `0` or `count: 0`.